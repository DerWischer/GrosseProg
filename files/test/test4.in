; Besser als 49
Jahrraum
Wautoa
Baumku
Hrswats
umilue
hahntee
NDART