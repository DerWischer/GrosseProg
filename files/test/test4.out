; BESSER ALS 49

Eingelesene Wörter: 
JAHRRAUM, WAUTOA, BAUMKU, HRSWATS, UMILUE, HAHNTEE, 
NDART

Rätsel nicht versteckt
     N    
     D    
   W A B  
  JAHRRAUM
   URT UM 
   TS  MI 
   OW  KL 
   AA  UU 
HAHNTEE E 
    S     

Rätsel versteckt
UPOOXNOOJK
PRRWPDGIZF
CHYWMAMBAY
YZJAHRRAUM
WBTURTFUMO
VQSTSREMIG
CPLOWWNKLP
CDSAACFUUY
HAHNTEEAES
ALDFSSTXOT

Kompaktheitsmaß: 100
